### ISSUE TITLE
SMS auto replies are truncated to a single SMS

### ISSUE THREAD

**Issue by abbyad** [medic/medic-webapp#3059]
Some outgoing messages are truncated to fit into a single SMS.

This is a placeholder to characterize the behaviour of SMS sent from different parts of the app/config, and then we can decide how to proceed.

**Comment by abbyad** [comment]
# Sentinel
## All test messages sent created via a Sentinel transition were truncated to 160 chars:
![image](https://cloud.githubusercontent.com/assets/789512/22348535/02f08eb4-e3da-11e6-8956-07a76e15c13f.png)

# Messages tab
## Sending from the Messages tab has no limit single message limit:
![image](https://cloud.githubusercontent.com/assets/789512/22348590/419b84e8-e3da-11e6-9a33-ed73b4658d8a.png)

## ...but perhaps should have a limit of some kind:
![image](https://cloud.githubusercontent.com/assets/789512/22348619/67d46468-e3da-11e6-814b-a84aba3fb4bf.png)

**Comment by abbyad** [comment]
With new projects we are giving the nurse the option to write additional notes for a CHWs, and these are sent to them along with the "autoreply" of that form. It would be useful to have the option to send longer messages, and leave it up to the configurer to try to fit the contents into a single message as needed.

**Comment by abbyad** [comment]
Closing and we can create a new issue for the feature request.

**Comment by estellecomment** [comment]
Trying to understand this better @abbyad , a couple questions.

For sentinel messages being truncated, it's not just the UI page displaying `...`, the SMS are actually truncated, right? (sorry, maybe a stupid question, but thought I'd check)

Did you try for all types of SMS messages? From [this list](https://github.com/medic/medic-webapp/issues/2487#issuecomment-242805025) : Registrations, Schedules, Patient Reports, Notifications, Alerts, Scheduled Report Reminders.

Are we already allowing nurses to send multiple-length auto-replies? In which case the feature is already in use? Or is it a feature request? (confused here)

**Comment by abbyad** [comment]
@estellecomment, it is truncated in the db as well, so it's not just hiding overflow:

![image](https://cloud.githubusercontent.com/assets/789512/22372012/91e93d6c-e468-11e6-9271-8b512350fb52.png)

I'll mention it in #3083 as well, but it looks like it gets truncated [here](https://github.com/medic/medic-sentinel/blob/227f1a870e555b1cfeb85cd625e5c8468c6a84f7/lib/utils.js#L129-L132) in Sentinel.

**Comment by abbyad** [comment]
Ok, that was a terrible example screenshot because futon does hide part of the string in that view. Here is it in full:
![image](https://cloud.githubusercontent.com/assets/789512/22375093/b877a21c-e476-11e6-8cf3-eecbd797a4ca.png)

In terms of if it is a feature request or a bug, many projects already use variables such as CHW or patient name in the responses, so those with names that are too long will have the messages truncated. As @browndav said `IMO silent truncation = data loss = bug`
